### DISCUSSION TITLE
How to use it with GraphQL?

### DISCUSSION THREAD

**Discussion by frederikhors** [BetterTyped/hyper-fetch#49] [Q&A] [answered]
I cannot find any example / docs / issue / discussion on using it with a GraphQL endpoint.

Can you help me understand?

**Comment by GerasNyx** [comment, marked as answer]
@frederikhors  - Sorry for the long silence. Lately we've introduced completely new GraphQL adapter - https://hyperfetch.bettertyped.com/docs/documentation/adapters/graphql/introduction/ - hope this doc section will help you :)

**Reply by frederikhors** [reply]
Thank you. I would like to read an example with Svelte too (I think today there isn't an adapter, yet, right? So I think we should use `core`).

Is there any example?

**Reply by GerasNyx** [reply]
There is no dedicated adapter for svelte, However using the `core` should be more than enough :)

**Reply by prc5** [reply]
@frederikhors here it is https://hyperfetch.bettertyped.com/docs/documentation/adapters/graphql/introduction/
